app/decorators/user_decorator.rb

```diff
@@ -51,6 +51,13 @@ def active_or_pending_profile
     user.active_profile || pending_profile
   end
 
+  # This user's most recently activated profile that has also been deactivated
+  # due to a password reset, or nil if there is no such profile
+  def password_reset_profile
```

Comment:
that's a good question. There might be legitimate reasons for multiple `password_reset_profile` records, but we should probably disallow more than one `pending_profile` record at at time, or at least with the same PII. Maybe detect that at proofing time ("You have a pending profile -- enter the OTP and click here to resend the letter")?